src/Components/App.js

```diff
@@ -1,13 +1,16 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Map from './Map'
+import Tile from './Tile'
+import Player from './Player'
 import '../Styles/App.css'
 
 const App = () => {
 
```

Comment:
We might be able to have the App component take on some of the data management by accessing files or helper components if they don't need to change often and just need to be handed down.